java/cdk-environment/src/main/java/sleeper/environment/cdk/networking/NetworkingStack.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.environment.cdk.networking;
+
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.ec2.*;
+import software.constructs.Construct;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+public class NetworkingStack extends Stack {
+
+    private final Vpc vpc;
+
+    public NetworkingStack(Construct scope, StackProps props) {
+        super(scope, props.getStackName(), props);
+
+        vpc = Vpc.Builder.create(this, "Vpc")
+                .cidr("10.0.0.0/16")
```

Comment:
Instance param?

Reply:
I would expect if you care about the specific configuration you'd probably rather set it up yourself. This is just intended to give a suitable environment to deploy Sleeper in if you don't already have one.

Reply:
I think the only reason you'd care about that is for mapping between IPs in different VPCs? Don't you need a mapping for that anyway?